### DISCUSSION TITLE
Problema no aplicativo de música áudio baixo

### DISCUSSION THREAD

**Discussion by claudioabreu07** [elementary/music#776] [Q&A]
Olá pesseal, tudo bem ? 
Quero reportar um problema de áudio do aplicativo de música nativo do elementary.  Com a última atualização, mesmo deixando o volume 100%, não corresponde. Fiz todos os ajustos de configurações e não resolveu.  Alguem, mais  com esse mesmo problema??

**Comment by zeebok** [comment]
I had something similar on a fresh install, when I would try to use Music I couldn't hear anything. It turns out that the system muted the application for some reason and I just needed to turn it up. I used Mixer in the AppCenter to adjust the volume. Maybe it is similar for you?
